src/materials/CrazyKCPlantFits.C

```diff
@@ -15,6 +15,7 @@ InputParameters
 CrazyKCPlantFits::validParams()
```

Comment:
Is it worth renaming these classes to use a more generic name? If yes, let's make a separate issue